app/gui2/src/components/GraphEditor.vue

```diff
@@ -394,13 +394,18 @@ function showComponentBrowser(nodePosition?: Vec2, usage?: Usage) {
   componentBrowserVisible.value = true
 }
 
-function startCreatingNodeFromButton() {
+function addNodeFromSelection() {
   const targetPos =
     placementPositionForSelection() ??
     nonDictatedPlacement(DEFAULT_NODE_SIZE, placementEnvironment.value).position
   showComponentBrowser(targetPos)
 }
```

Comment:
Technically, it is not only about selection, but I should be able to add a node even without any selected nodes (the position should be the scene origin, pushing down if occupied).